From d00998ccbeb4fa3044c834e386e0df61cf7a63fa Mon Sep 17 00:00:00 2001
From: Rushan Suleymanov <rushans@google.com>
Date: Wed, 15 Jan 2025 14:44:15 -0800
Subject: [PATCH] [M132] Remove DumpWithoutCrash on creation empty local tabs

Because crash reporting cant handle the amount of traffic generated by this Dump Without Crashing, we are disabling it on M132.

Bug: 378259860, 389098358, 388826613
Change-Id: Id60acf8874545b91b8f65a4c788d6560f8edd25b
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/6164407
Commit-Queue: David Pennington <dpenning@chromium.org>
Commit-Queue: Krishna Govind <govind@chromium.org>
Reviewed-by: Krishna Govind <govind@chromium.org>
Cr-Commit-Position: refs/branch-heads/6834@{#3802}
Cr-Branched-From: 47a3549fac11ee8cb7be6606001ede605b302b9f-refs/heads/main@{#1381561}
---

--- a/components/saved_tab_groups/internal/stats.cc
+++ b/components/saved_tab_groups/internal/stats.cc
@@ -191,14 +191,6 @@
 
   base::UmaHistogramBoolean("TabGroups.Sync.AddedGroupIsEmptyLocally",
                             group.saved_tabs().empty());
-  if (group.saved_tabs().empty()) {
-    // This seems like it may be happening in production, unexpectedly; it
-    // should still be rare, so add a diagnostic dump to find out if anyone is
-    // adding empty groups, so we can see if we need to support it properly or
-    // if this should become a CHECK. In the latter case we'll need to update
-    // the many tests which do add empty groups.
-    base::debug::DumpWithoutCrashing();
-  }
 }
 
 void RecordEmptyGroupsMetricsOnGroupAddedFromSync(const SavedTabGroup& group,
